MiniApp/Classes/core/MASDKError.swift

```diff
@@ -75,6 +78,9 @@ extension MASDKError: LocalizedError {
             return "error_miniapp_id_not_found".localizedString()
         case .metaDataFailure:
             return "error_miniapp_meta_data_required_permissions_failure".localizedString()
+        case .failedToConformToProtocol:
+            return "Host app failed to implement required interface".localizedString()
```

Comment:
This should be `failed_to_conform_to_protocol`

Reply:
oh nice one, I'll fix this

Reply:
done